### DISCUSSION TITLE
[REQUEST] Can you please support OrangePi 3b

### DISCUSSION THREAD

**Discussion by defencedog** [7Ji/orangepi5-archlinuxarm#50] [Q&A] [answered]
Can you please provide an image for the stated device. Or at least provide information / steps to achieve this by self endeavour

**Comment by 7Ji** [comment, marked as answer]
Start by reading through and understanding the source code of 
- kernel building: https://github.com/7Ji-PKGBUILDs/linux-aarch64-rockchip-rk3588-bsp5.10-orangepi-git
- fip building: https://github.com/7Ji/orangepi5-rkloader
- image building: https://github.com/7Ji/orangepi5-archlinuxarm

No image would be provided if one with their own 3b doesn't bother to tinker with kernel, fip and images by themselves first
